Source/WebKit/_WebKit_SwiftUI/API/WebView.swift

```diff
@@ -152,4 +185,25 @@ extension WebView {
         /// The URL of the link that the user clicked.
         public let linkURL: URL?
     }
- }
+}
+
+extension WebView {
+    private enum Storage: DynamicProperty {
+        case state(State<WebPage>, URL?)
+        case webPage(WebPage)
+
+        var webPage: WebPage {
+            switch self {
+            case let .state(state, _): state.wrappedValue
+            case let .webPage(webPage): webPage
+            }
+        }
+
+        var url: URL? {
```

Comment:
Not `public`, so this is less impactful, but I do worry this might overlaps a little bit with “the webpage’s current url” just from it’s name (when people read the name that is). It’s also incorrect to call this `initialURL` given the behaviors here though, so I’m not exactly sure how to clarify it? ~~`preferredURL`~~?